### PR TITLE
Fraunces and Space Grotesk designers bio added

### PR DESCRIPTION
**Pull request by vv-monsalve**
This PR includes:

- [x] Fraunces designers
catalog/designers/undercasetype
catalog/designers/phaedracharles
catalog/designers/flaviazimbardi

- [x] Space Grotesk designer
catalog/designers/floriankarsten

- [x] Gayathri designers info
catalog/designers/smc
catalog/designers/binoydominic

- [x] Manjari designer info
catalog/designers/santhoshthottingal

### CODE REVIEW

**Comment by RosaWagner** [comment]
Can you confirm this ? It doesn't look consistent:

Capture d’écran 2021-02-16 à 13 06 22

**Comment by vv-monsalve** [comment]
> Can you confirm this ? It doesn't look consistent:

It isn't. The file was ok in my local files but not in upstream repo somehow. Updated.

File: catalog/designers/floriankarsten/bio.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<p>Florian Karsten Studio (Brno, Czech Republic) focuses on graphic design, type design and programming. We create websites, books, programmes, typefaces and above all, functional systems. We’re excited about open-source and peer2peer networks.</p>
```
Comment (RosaWagner) [inline catalog/designers/floriankarsten/bio.html:1]: I suggest not using "we" but "they"

Comment (vv-monsalve) [reply catalog/designers/floriankarsten/bio.html]: Yes, something sounded a bit strange in that "we". Changed

**Comment by RosaWagner** [comment]
Just saw an error : phaedra charles has a jpeg image, last it blocked the push, needs to be png (and I think Marc corrected the image path in the info.pb, so it has to be updated too when the format is corrected).

We really need more FB check for the designer profile.

**Comment by vv-monsalve** [comment]
Sigh. I try to extra-check every detail :-/ So yes having FB fail would be of great help.
Also, we could create a form to ask for this material with restrictions on the length and file format.

FIx done in #3110

**Comment by RosaWagner** [comment]
> Also, we could create a form to ask for this material with restrictions on the length and file format.

Yep I just did it this morning, we'll review it on thursday

**Comment by RosaWagner** [comment]
Undercase Type info.pb has the wrong name

**Comment by RosaWagner** [comment]
Binoy Dominic not appearing in designers of Gayathri.

Note: I think it would be better to separate PRs cause it is checking and tracking confusing and complicated when several fonts designers are concerned in one PR.

**Comment by vv-monsalve** [comment]
> Florian Karsten has the description of Undercase Type

Don't know why if it was ok in [this commit](https://github.com/google/fonts/pull/3093/commits/0c4367129d95c7536667e88cee3db4aec71fd633)

**Comment by vv-monsalve** [comment]
Mhm, that makes sense. I'm fixing everything now. How would it be better to fix this, a different PR for each? I did this multiple designers PR precisely trying to avoid extra PR revision processes, but I can now see it would be clearer to have one PR per designer or project for regular additions to the catalog.

**Comment by RosaWagner** [comment]
I agree, I would go for 1 PR per directory added/modified, cause that is how we notify the eng team, it would make more sense

**Comment by vv-monsalve** [comment]
Confirming: then 1 PR per designer, not project, right? E.g. Flavia PR 1, Phaedra PR 2, Undercase PR 3. 

> I agree, I would go for 1 PR per directory added/modified, cause that is how we notify the eng team, it would make more sense

**Comment by RosaWagner** [comment]
Yes, let's test it like that for few weeks and see if it makes our life better or not.

**Comment by vv-monsalve** [comment]
I wonder if it would imply extra time/efforts as it would require like a branch per each to avoid commits conflicts.
A middle point could be 1 PR per project.

**Comment by vv-monsalve** [comment]
> Undercase type has Florian's name in info.pb

Fixed in #3168

**Comment by vv-monsalve** [comment]
> Binoy Dominic not appearing in designers of Gayathri.

Binoy Dominic added to Gayathri's Metadata in #3169

**Pull request merged**